docs/baseline.md

```diff
@@ -176,5 +179,6 @@ Here are some areas for future consideration and not currently in-scope for Base
 * Progressive enhancement safe (i.e., limited penalties for support failures)
 * Developer feedback requested
 * Buggy (e.g., supported but in ways that are surprising and semi-interoperable)
+* Buggy or unsupported in assistive tools
```

Comment:
Because this is a future consideration and Baseline is not going to add AT support, I suggest something a bit more straightforward as a future goal (since it keeps the scope strictly with browsers and does not require AT):

"Confirm if features are exposed to platform accessibility APIs (AAPIs) as specified."